### ISSUE TITLE
stuff to use kubernaut for demo

### ISSUE THREAD

**Issue by rhs** [datawire/kubernaut#17]
1. ability to control the lifespan of the cluster (hard limit of 1 hour no good for demo)
2. ability to have multiple clusters (I don't want my long lived cluster to prevent me from getting shorter lived ones)
3. some way to specify a stable dns name for my cluster when I claim it (we put this in slides prior to demo so audience can participate)
4. LoadBalancer support
5. Multiple nodes (more capacity)

**Comment by plombardi89** [comment]
@richarddli can you expand on the specifics of that?

**Comment by richarddli** [comment]
i want to be able to do:

```
kubernaut claim
git clone github.com/datawire/todo
cd todo
forge deploy
```

and be able to have a working todo app.

**Comment by plombardi89** [comment]
Right... I am asking what doesn't work about that specifically? I mean I can go off and test it myself and maybe I'll hit an issue but it might also not be the same issue as yours.

I'm trying to figure out what feature(s) or bug is preventing that from being possible right now. There's prioritization and effort concerns also as I have a hunch I know what one of the issues is.

**Comment by richarddli** [comment]
For starters, it doesn't give me an IP address for the API Gateway because it's a LoadBalancer.

Second, I see this:

Tolerations:	<none>
Events:
  FirstSeen	LastSeen	Count	From			SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----			-------------	--------	------			-------
  1m		10s		8	{default-scheduler }			Warning		FailedScheduling	PersistentVolumeClaim is not bound: "todo-db-persistent-storage-todo-db-0"

**Comment by plombardi89** [comment]
Thanks! Those are at actionable items.
